lib/views/activites/count_activity_view.dart

```diff
@@ -46,7 +46,7 @@ class _CountActivityViewState extends State<CountActivityView> {
             Padding(
               padding: const EdgeInsets.all(12.0),
               child: widget.isReview
-                  ? BodyText(
+                  ? Body(
```

Comment:
Not really a fan as the name Body doesn't clearly identify what this does

Reply:
Whereas 'Heading' and 'Sub-heading' is already clear

Reply:
Yea I realise that I can change it back if we can't come up with a better name

Reply:
I think we should change it back